content/_help/manage-your-account/personal-key._es.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: help
+title: Clave personal 
+category: trouble-signing-in
+permalink: /es/help/manage-your-account/personal-key/
+order: 3 
+redirect_from:
+  - /es/es/help/manage-your-account/personal-key/
```

Comment:
Should this path have two `/es/es` in it?

Reply:
Nope.